argilla/docs/reference/argilla/datasets/dataset_records.md

```diff
@@ -193,6 +194,54 @@ Records can also be updated using the `log` method with records that contain an
     1. In this example, the Hugging Face dataset matches the Argilla dataset schema.
     2. The `uuid` key in the Hugging Face dataset corresponds to the `id` field in the Argilla dataset.
 
+### Adding and updating records with images
+
+Argilla datasets can contain image fields. You can add images to a dataset by passing the image to the record object as either a remote URL, a local path to an image file, or a PIL object. The field names must be defined as an `rg.ImageField` in the dataset's `Settings` object to be accepted. Images will be stored in the Argilla database and returned as rescaled PIL objects.
+
+=== "From a data structure with local file paths"
```

Comment:
We don't show a flow for PIL objects currently.

Reply:
cc @burtenshaw Apart from the note, we could specify the behavior here too

Reply:
@davidberenstein1957 I don't think that's necessary because the API reference for `Record` says that the `fields` param is a dict that can contains PIL. I also wouldn't encourage that route because the other two options are more common and ideal:

**should example**
- data structure with paths
- hf datasets with encoded PIL

**should not show example for**
- data structure with PIL
- iterator and Record init with PIL

Reply:
@burtenshaw, makes sense. In that case we should update the documentation to avoid mentioning the specific PIL-based approach too.

Reply:
I think we can mention it but not supply a snippet? The fields param of `Record` is not super straightforward because it's a dict.

Reply:
@burtenshaw, I think we can but I think it might lead to confusion and people asking about it.